bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BKException.java

```diff
@@ -72,6 +72,8 @@ public static BKException create(int code) {
             return new BKNotEnoughBookiesException();
         case Code.NoSuchLedgerExistsException:
             return new BKNoSuchLedgerExistsException();
+        case Code.NoSuchLedgerExistsOnMetadataServerException:
```

Comment:
nit: I think we used `MetadataStore` or `MetadataService` in other places. so it might be worth to use similar name here.